pkg/operator/nfs/server.go

```diff
@@ -67,8 +67,8 @@ func Run(ganeshaConfig string) error {
 	// Start ganesha.nfsd
 	logger.Infof("Running NFS server!")
 	cmd := exec.Command("ganesha.nfsd", "-F", "-L", ganeshaLog, "-f", ganeshaConfig, "-N", ganeshaOptions)
-	if out, err := cmd.CombinedOutput(); err != nil {
-		return fmt.Errorf("ganesha.nfsd failed with error: %v, output: %s", err, out)
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("ganesha.nfsd failed with error: %v", err)
```

Comment:
since we already set debug level to `NIV_INFO`, i think it's safe now to use `CombineOutput` function

Reply:
@aberfeldy can you please address this one?